package/MDAnalysis/analysis/contacts.py

```diff
@@ -364,14 +367,22 @@ class Contacts(AnalysisBase):
 
     Attributes
     ----------
-    timeseries : list
-        list containing *Q* for all refgroup pairs and analyzed frames
+    results.timeseries : numpy.ndarray
+        2D array containing *Q* for all refgroup pairs and analyzed frames
+
+    timeseries : numpy.ndarray
+        Deprecated alias to :attr:`results.timeseries`. Will be removed in
+        MDAnalysis 3.0.0.
+
```

Comment:
```suggestion
        MDAnalysis 3.0.0.
        
        .. deprecated:: 2.0.0
           The :attr:`timeseries` attribute is deprecated in favour of
           :attr:`results.timeseries`, it will be removed in MDAnalysis 3.0.0.
           
```